### DISCUSSION TITLE
iOSでの認証時にURLスキーマ周りでエラーが発生する

### DISCUSSION THREAD

**Discussion by noppefoxwolf** [pixiv/vroid-sdk-developers#18] [Q&A]
旧コミュニティで話されていた問題ですが、SDKの対応予定などはありますか？
https://spectrum.chat/vroid-developers/general/ios-url~892cd63b-375a-47b0-8241-5e2a664aa9f1

**Comment by Haresaku** [comment]
はい。今後のアップデートでこちらの問題については対応する予定です。
ご不便おかけしますが、今しばらく旧コミュニティサイトで共有させていただいているワークアラウンドで対応していただけますでしょうか
